index.html

```diff
@@ -201,7 +204,14 @@ <h4>Skills</h4>
     <h1>Get Started</h1>
     <p>If you have an application you are interested in developing, a feature that you need to build or a project that needs coding. I’d love to help with it</p>
     <div class="bot">
-        <!-- forms here  -->
+        <form action="https://formspree.io/f/xlezzvld" method="post">
+            <input type="text" name="name" id="name" placeholder="your name" maxlength="30" required>
+            <input type="email" name="email" id="email" placeholder="your email" required>
+            <textarea name="text" id="text" cols="20" rows="8" placeholder="Enter your message here..." maxlength="500" required></textarea>
+            <div class="bot">
+                <button type="button" id="bomm">Start Collaboration</button>
```

Comment:
- [OPTIOANAL ]  I am grateful that you've managed to implement most of the changes requested by previous reviewers, but again your form submission is not working. When I Click start collaboration, there is nothing sent. because I am supposed to see a successful message telling me that I sent the information. so as the first review asked, configured it correctly . 
![image](https://user-images.githubusercontent.com/43877689/171385206-5681f81d-6f27-4b3f-aaba-ceaac911f1c5.png)
Here is the issue causing your form not to work 
![image](https://user-images.githubusercontent.com/43877689/171385888-d6d07aeb-0389-4dcc-a470-ea399b4b1245.png)
Your button is supposed to have type submit